addon/mixins/ajax-request.js

```diff
@@ -268,7 +268,13 @@ export default Mixin.create({
     const method = hash.method || hash.type || 'GET';
     const requestData = { method, type: method, url: hash.url };
 
-    if (isJSONAPIContentType(getHeader(hash.headers, 'Content-Type')) && requestData.type !== 'GET') {
+    if (
+      requestData.type !== 'GET'
```

Comment:
I should probably break it out into its own npm package. 😉 Sure, I'll inline it.